examples/simple-monorepo/README.md

```diff
@@ -1,6 +1,45 @@
-# ICMK Simple Example
+# IZE Simple Example
 
- ```shell script
-make infra
-make tunnel
+### Commands for generate terraform files
+
+```shell
+ize -c ./examples/simple-monorepo/.infra/env/testnut/ize.toml env terraform
```

Comment:
Let's just set the `IZE_CONFIG_FILE` env var to Ize config file one time in the very beginning, and not use `-c` flag anywhere below, sounds good?

Reply:
I want show what there is this  set config file method

Reply:
Yes, makes sense. In this specific  example's README we should skip it, I think, as it may confuse/scare the user a bit (as they need to set it every time). We should definitely keep it in the final docs.